### PR TITLE
Feat/kyc additional requirements

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: src/screens/marketplaces/mkpSlice.js [hunk 1]
```diff
@@ -399,6 +400,20 @@ export const mkpOperations = {
 		dispatch(mkpActions.setShowPaymentProgressModal(true));
 		dispatch(mkpActions.setPaymentInProgress(true));
 	},
+	submiAdditionalInformation: () => async (dispatch, getState) => {
```
Comment (andregoncalves) [inline src/screens/marketplaces/mkpSlice.js:403]: Should be `submitAdditionalInformation`

File: src/screens/marketplaces/MarketplaceProductScreen/index.js [hunk 2]
```diff
@@ -37,7 +37,8 @@ export default function MarketplaceProductScreen(props) {
   const paymentInProgress = useSelector(mkpSelectors.getPaymentInProgress);
   const price = useSelector(mkpSelectors.getPrice);
   const handlePay = () => dispatch(mkpOperations.payApplication());
-
+  const handleAdditionalInfo = () => dispatch(mkpOperations.submiAdditionalInformation());
```
Comment (andregoncalves) [inline src/screens/marketplaces/MarketplaceProductScreen/index.js:40]: Should be `submitAdditionalInformation`

**Review by andregoncalves — APPROVED** [review]
Looks good!

File: src/screens/marketplaces/mkpSlice.js (hunk 1 above)
Comment (maycon-mello) [reply src/screens/marketplaces/mkpSlice.js:403]: Thank, I update that.

File: src/screens/marketplaces/MarketplaceProductScreen/index.js (hunk 2 above)
Comment (maycon-mello) [reply src/screens/marketplaces/MarketplaceProductScreen/index.js]: Thanks, i fixed this one.

**Pull request merged**